src/frontend/Sider/Behandling/Inngangsvilkår/Inngangsvilkår.tsx

```diff
@@ -22,22 +27,39 @@ const Container = styled.div`
 `;
 
 const Inngangsvilkår = () => {
+    const { request } = useApp();
+    const { behandling } = useBehandling();
     const { regler, hentRegler } = useRegler();
     const { vilkårsvurdering } = useVilkår();
 
+    const [vilkårperioder, settVilkårperioder] = useState<Ressurs<Vilkårperioder>>(
+        byggTomRessurs()
+    );
+
     useEffect(() => {
         hentRegler();
     }, [hentRegler]);
 
+    const hentVilkårperioderCallback = useCallback(() => {
+        request<Vilkårperioder, null>(`/api/sak/vilkarperioder/${behandling.id}/periode`).then(
+            settVilkårperioder
+        );
+    }, [request, behandling.id]);
+
+    const hentVilkårperioder = useRerunnableEffect(hentVilkårperioderCallback, [behandling.id]);
+
     return (
         <Container>
             {!erProd() && <FyllUtVilkårKnapp />}
-            <DataViewer response={{ regler, vilkårsvurdering }}>
-                {({ regler, vilkårsvurdering }) => (
+            <DataViewer response={{ regler, vilkårsvurdering, vilkårperioder }}>
+                {({ regler, vilkårsvurdering, vilkårperioder }) => (
                     <>
                         {features.nyeInngangsvilkår && (
-                            <InngangsvilkårProvider>
-                                <InngangsvilkårInnhold regler={regler.vilkårsregler} />
+                            <InngangsvilkårProvider
+                                vilkårperioder={vilkårperioder}
+                                hentVilkårperioder={hentVilkårperioder}
+                            >
+                                <InngangsvilkårInnhold />
```

Comment:
Det er ikke lenger å ha en egen fil for `InngangsvilkårInnhold`, tenker dere dette bør flyttes ut hit? Da hentes også `stønadsperioder` her ute. Evt. kan henting av vilkårsperioder flyttes inn. 

Dersom vilkår for `PassBarn` flyttes til en egen side vil det ikke være nødvendig å hente `regler` og `vilkårsvurdering` her etterhvert

Reply:
- [Flytte inn i innhold ](https://github.com/navikt/tilleggsstonader-sak-frontend/commit/be7630f784acb5fd56e0f7fc8f265aa71b8aad81)
- [Flytte innhold ut](https://github.com/navikt/tilleggsstonader-sak-frontend/commit/7298a99b03149155fbe3c9c7e6d4c6dbb6c1850d) (tror jeg liker denne best, hvertfall om resten blir borte)

Reply:
Sista blir nog fint 👍